src/charging-station/ui-server/ui-services/AbstractUIService.ts

```diff
@@ -18,7 +18,7 @@ export default abstract class AbstractUIService {
   constructor(uiServer: AbstractUIServer) {
     this.uiServer = uiServer;
     this.messageHandlers = new Map<ProtocolCommand, ProtocolRequestHandler>([
-      [ProtocolCommand.LIST_CHARGING_STATIONS, this.handleListChargingStations.bind(this)],
```

Comment:
Please keep the binding to this for performance reason. You can cast if needed to silence the linter.